content/docs/command-reference/plots/index.md

```diff
@@ -125,8 +125,8 @@ All metrics files given to `dvc plots show` and `dvc plots diff` as input are
 combined together into a single data array for injection into a template file.
 There are two important fields that DVC adds to the plot data:
 
-- `index` - self-incrementing, zero-based counter for the data rows/values. In
-  many cases it corresponds to a machine learning training epoch or step number.
+- `index` - zero-based counter for the data rows/values. In many cases it
```

Comment:
not related? please let's not include it in the PR

also, not sure even about the reason of changing this

Reply:
For consistency, to match the wording in the text about indices in complex foreach groups found [here](https://github.com/iterative/dvc.org/pull/2052/files#diff-584bff5a435baee84112ffda33bc2d9dc94e1512e7306322f43655d5e7593876R401-R402)

Reply:
hmm, why should the match at all? those are two completely different features?

`self-incrementing` was an important part here to my mind, btw

Reply:
It's just nice that they match. But I don't have a strong opinion on this. Reverted.

Reply:
> self-incrementing was an important part

Actually I'm not sure that's what self-incrementing means though. Let's discuss in https://github.com/iterative/dvc.org/pull/2062#pullrequestreview-558981991?